lxc/info.go

```diff
@@ -642,6 +643,50 @@ func (c *cmdInfo) instanceInfo(d lxd.InstanceServer, name string, showLog bool)
 				row = append(row, "NO")
 			}
 
+			// Display attached volume snapshots
+			var attachedVolumeSnapshots string
+			if snap.Config["volatile.attached_volumes"] != "" {
+				// Parse the JSON map of volume UUID -> snapshot UUID
+				var attachedVolumeUUIDs map[string]string
+				err := json.Unmarshal([]byte(snap.Config["volatile.attached_volumes"]), &attachedVolumeUUIDs)
+				if err != nil {
+					return err
+				}
+
+				// Find the volume names and snapshot names from the UUIDs
+				volumeSnapshotNames := make([]string, 0, len(attachedVolumeUUIDs))
+				for volUUID, snapUUID := range attachedVolumeUUIDs {
+					for _, dev := range inst.Devices {
```

Comment:
should this be snap.Devices (the volume snapshot might be of a disk device that is no longer attached to the instance)?